File: client/Client/Network/Messages/Session/SelectSectionMessage.cs
using LoadRunnerClient.DTOs;
using System.Collections.Generic;

namespace LoadRunnerClient.Messages
{
    public class SelectSectionMessage
    {
        public const string TYPE = "SelectSectionMessage";
        private int _section;
        public int section { get => _section; set => _section = value; }

        private Dictionary<int, List<string>> _sections;
        public Dictionary<int, List<string>> sections { get => _sections; set => _sections = value; }

        public SelectSectionMessage()
        {
        }

        public SelectSectionMessage(int sec)
        {
            this.section = sec;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/session/SelectSectionMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.session;

import java.util.HashMap;
import java.util.List;

public class SelectSectionMessage {
	public static final String TYPE = "SelectSectionMessage";
	private int section;
	private HashMap<Integer,List<String>> sections;

	public SelectSectionMessage() {
	}

	public SelectSectionMessage(int x) {
		this.section = x;
	}

	public SelectSectionMessage(HashMap<Integer,List<String>> list){
		this.sections = list;
	}

	public int getSection() {
		return section;
	}

	public void setSection(int section) {
		this.section = section;
	}

	public HashMap<Integer,List<String>> getSections() {
		return sections;
	}

	public void setSections(HashMap<Integer,List<String>> sections) {
		this.sections = sections;
	}
}
